### PR TITLE
Bugfix/os and programming language in upload solution is not submitted

### PR DESCRIPTION
**Pull request by lorenzschreyer**

### CODE REVIEW

File: backend/codingchallenge/codingchallenge/settings.py
Comment (HeiligeMeier) [inline backend/codingchallenge/codingchallenge/settings.py:1]: Don't push comments

Comment (lorenzschreyer) [reply backend/codingchallenge/codingchallenge/settings.py:1]: In this case i think its Not bad to have the pls database commented for Testung purposes without running docker

**Review by dak2791 — APPROVED** [review]

File: backend/codingchallenge/codingchallenge/settings.py
Comment (HeiligeMeier) [reply backend/codingchallenge/codingchallenge/settings.py:1]: you could use this instead

if 'test' in sys.argv: 
    Databases = {
        ...
    }
else: 
    Databases = {
        ...
    }

Comment (lorenzschreyer) [reply backend/codingchallenge/codingchallenge/settings.py:1]: @HeiligeMeier No actually not. We had to test the upload behavior of an additional file. So we had to edit the database after every file upload to ensure the status and submission_time is being reset. This was much more convenient with the file based database approach. @KRIYO144

**Review by HeiligeMeier — APPROVED** [review]

**Pull request merged**